src/hooks/useUserMarketInfo.ts

```diff
@@ -141,10 +141,10 @@ const useUserMarketInfo = ({
   const [totalBorrowBalance, totalBorrowLimit] = assetList.reduce(
     (acc: [BigNumber, BigNumber], asset: Asset) => {
       const borrowBalanceUSD = asset.borrowBalance.times(asset.tokenPrice);
-      acc[0].plus(borrowBalanceUSD);
+      acc[0] = acc[0].plus(borrowBalanceUSD);
       if (asset.collateral) {
         const supplyBalanceUSD = asset.supplyBalance.times(asset.tokenPrice);
-        acc[1].plus(supplyBalanceUSD.times(asset.collateralFactor));
+        acc[1] = acc[1].plus(supplyBalanceUSD.times(asset.collateralFactor));
       }
       return acc;
     },
```

Comment:
You also need to update line `153` so `userVaiMinted` is added to `totalBorrowBalance`